doc/stages/readers.ept.rst

```diff
@@ -44,6 +42,11 @@ resolution
 
     The resulting resolution may not be exactly this value: the minimum possible resolution that is at *least* as precise as the requested resolution will be selected.  Therefore the result may be a bit more precise than requested.
 
+addons
```

Comment:
You do not provide an example in the reader documentation, instead saving it for the writer. I understand it may be redundant to show it both places. Maybe a cross reference? I don't know.

Reply:
Added an addon example to the reader in 59f40a0, deferring to the writer doc for details.